### PR TITLE
Private/anirudh/emp yaml temp

### PR DESCRIPTION
**Pull request by AnirudhPokala**
Added EMP sample yamls templates simple combined template, individual objects templates. 
https://platform9.atlassian.net/browse/EMP-1091

### CODE REVIEW

File: emp/emp-yamls/02_eks-cluster.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+apiVersion: emp.pf9.io/v1alpha1
+kind: EKSCluster
+metadata:
+  name: emp-eks-cluster
+spec:
+  # EKS cluster name as in AWS
+  clusterName: emp-eks-cluster
+  
+  # AWS region where the EKS cluster is running
+  clusterRegion: us-east-1
```
Comment (OmkarDeshpande7) [inline emp/emp-yamls/02_eks-cluster.yaml:10]: Please remove hardcoded values

File: emp/emp-yamls/02_eks-cluster.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  
+  # AWS Cloud Credentials reference through which we can access this EKS cluster.
+  cloudProviderRef:
+    apiGroup: emp.pf9.io/v1alpha1
+    kind: AWSCredentials
+    name: aws-qa-creds
+  
+  # EKS Cluster Security groups ID (list)
+  securityGroups:
+    - "sg-01ec63883149b003d"
```
Comment (OmkarDeshpande7) [inline emp/emp-yamls/02_eks-cluster.yaml:20]:
```suggestion
    - "sg-012345xxxxxxxx"
```

File: emp/emp-yamls/04_baremetalpool.yaml [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    azs:
+      - us-east-1a
+      - us-east-1b
+      - us-east-1c
+    
+    # Network configuration for baremetal pool
+    network:
+      # AWS VPC to consider to host baremetal machines.
+      vpc:  
+        id: vpc-0ee66d01a874305d5
```
Comment (OmkarDeshpande7) [inline emp/emp-yamls/04_baremetalpool.yaml:32]:
```suggestion
        id: vpc-0eexxxxxxxxxx
```

File: emp/emp-yamls/04_baremetalpool.yaml [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+        # public NAT subnet
+        - id: subnet-0c0127713bbab8ad1
+          isPrivate: false
+          az: us-east-1a
+        - id: subnet-03fc122ea4adb66f5
+          isPrivate: false
+          az: us-east-1b
+        - id: subnet-0128ed1d460b8b761
+          isPrivate: false
+          az: us-east-1c
```
Comment (OmkarDeshpande7) [inline emp/emp-yamls/04_baremetalpool.yaml:55]: please truncate all hardcoded subnet names

File: emp/emp_objects_template.yaml
Comment (OmkarDeshpande7) [inline emp/emp_objects_template.yaml:1]: same for all comments combined from smaller files.

File: emp/emp-yamls/02_eks-cluster.yaml (hunk 1 above)
Comment (AnirudhPokala) [reply emp/emp-yamls/02_eks-cluster.yaml:10]: Its a sample template only.. just added for reference..

File: emp/emp-yamls/04_baremetalpool.yaml (hunk 4 above)
Comment (AnirudhPokala) [reply emp/emp-yamls/04_baremetalpool.yaml:55]: Those all are dummy values..

File: emp/emp_objects_template.yaml
Comment (AnirudhPokala) [reply emp/emp_objects_template.yaml:1]: Those all are dummy sample values.. not actual values.

**Review by OmkarDeshpande7 — APPROVED** [review]
lgtm

File: emp/emp-yamls/02_eks-cluster.yaml (hunk 1 above)
Comment (AnirudhPokala) [reply emp/emp-yamls/02_eks-cluster.yaml:10]: Added a comment with support region list..

**Pull request merged**